src/ralph/networks/admin.py

```diff
@@ -208,6 +224,12 @@ def show_addresses(self, network):
     show_addresses.allow_tags = True
     show_addresses.short_description = _('Addresses')
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        qs = qs.annotate(ipaddress_count=Count('ips'))
+        qs = qs.extra(select={'subnetwork_count': 'rght - lft'})
```

Comment:
btw you could paste a link to the mptt here (https://github.com/django-mptt/django-mptt/blob/master/mptt/models.py#L594)